cmd/libs/go2idl/lister-gen/generators/expansion.go

```diff
@@ -41,10 +43,14 @@ func (g *expansionGenerator) Filter(c *generator.Context, t *types.Type) bool {
 func (g *expansionGenerator) GenerateType(c *generator.Context, t *types.Type, w io.Writer) error {
 	sw := generator.NewSnippetWriter(w, c, "$", "$")
 	for _, t := range g.types {
+		tags, err := util.ParseClientGenTags(t.SecondClosestCommentLines)
+		if err != nil {
+			fmt.Printf("Error parsing tags: %v", err)
```

Comment:
if we can't parse, our return code should be non-zero.  This seems to hide that.